### PR TITLE
Fix - Buttons on Android go up after closing keyboard

### PR DESCRIPTION
**Pull request by LendiHop**

### CODE REVIEW

File: src/components/token-icon/token-icon.styles.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ import { formatSize } from '../../styles/format-size';
 export const TokenIconStyles = StyleSheet.create({
   container: {
     marginVertical: formatSize(4),
-    marginHorizontal: formatSize(4)
+    marginHorizontal: formatSize(4),
+    borderRadius: formatSize(100),
+    overflow: 'hidden'
```
Comment (unixvb) [inline src/components/token-icon/token-icon.styles.ts:10]: why these changes are in this PR?

File: src/components/screen-container/screen-container.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export const ScreenContainer: FC<Props> = ({ isFullScreenMode = false, style, co
     <KeyboardAvoidingView
       style={[styles.scrollView, style]}
       keyboardVerticalOffset={keyboardVerticalOffset}
-      behavior="padding">
+      behavior="height">
```
Comment (andrewkryshtal) [inline src/components/screen-container/screen-container.tsx:25]: did you try this fix on iOS?

**Review by andrewkryshtal — APPROVED** [review]

File: src/components/screen-container/screen-container.tsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ export const ScreenContainer: FC<Props> = ({ isFullScreenMode = false, style, co
     <KeyboardAvoidingView
       style={[styles.scrollView, style]}
       keyboardVerticalOffset={keyboardVerticalOffset}
-      behavior="padding">
+      behavior={Platform.OS === 'ios' ? 'padding' : 'height'}>
```
Comment (unixvb) [inline src/components/screen-container/screen-container.tsx:25]: use `isIOS`

**Review by unixvb — APPROVED** [review]
